client/reader/following/main.tsx

```diff
@@ -71,6 +72,20 @@ function FollowingStream( { ...props } ) {
 					>
 						<ViewToggle />
 					</NavigationHeader>
+					<Banner
+						callToAction={ translate( 'Take the survey' ) }
```

Comment:
Since it leads to a separate site, I think we should have it open in a new window:


```suggestion
						target="_blank"
						callToAction={ translate( 'Take the survey' ) }
```